translations/scantailor_de.ts

```diff
@@ -1594,7 +1594,7 @@ p, li { white-space: pre-wrap; }
     <message>
         <location filename="../ui/RelinkingDialog.ui" line="97"/>
         <source>Undo</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Rückgangig</translation>
```

Comment:
Rückgängig machen?

I think that `type="unfinished"` should also be removed for new translations.